fontforge/python.c

```diff
@@ -9462,12 +9466,96 @@ static PyObject *PyFFGlyph_preserveLayer(PyFF_Glyph *self, PyObject *args) {
 Py_RETURN( self );
 }
 
-static PyObject *PyFFGlyph_BoundingBox(PyFF_Glyph *self, PyObject *UNUSED(args)) {
+static int *LayerArgToLayer(SplineFont *sf, PyObject* layerp) {
```

Comment:
Happily it appears we already use `-1` as an unknown/undefined layer number, so it should be safe here.  rather than having to do the two-level thing.

Reply:
Actually, ly_grid is `-1`, which is usually an invalid layer, but...it's still a layer. `ly_none` is `-3`, which I'll see about using instead right now.

Reply:
Ah, missed that in my poking around (-1 is used explicitly in some places), but, yes, `ly_none` is the way to go here.